fhd_core/setup_metadata/load_skyh5_source_catalog.pro

```diff
@@ -0,0 +1,33 @@
+FUNCTION load_skyh5_source_catalog, catalog
+  ; Restore a source catalog from a .skyh5 file
+  
+  file_id = H5F_OPEN(catalog)
+  dataset_id = H5D_OPEN(file_id, '/Data/stokes')
```

Comment:
Please add a comment to explain why you use `H5F_OPEN` at first, and `H5D_OPEN` later (f for file and d for data?)